doc/softphone.md

```diff
@@ -82,6 +82,69 @@ Once the client has a session, it can add event handlers for lower level control
 over sessions. `terminated` is an example of a session event;
 [all session events are detailed here].
 
+Once you have an initialized instance of the WebrtcSdk and events setup, you can create a softphone session in the following manner:
+
+``` ts
+await sdk.startSoftphoneSession({phoneNumber: '15555555555'});
+```
+ ## Softphone Session Methods
+
+ #### `startSoftphoneSession(params: IStartSoftphoneSessionParams)`
+ Creates a new softphone call with the given peer or peers.
+
+ Params:
+ * `params: IStartSoftphoneSessionParams` - Required: Contains the peers to start the session with. See interfaces below for more details regarding `IStartSoftphoneSessionParams`
+
+### Interfaces
+
+#### `IStartSoftphoneSessionParams`
+Contains the peer information for the outbound call.
+
+```ts
+interface IStartSoftphoneSessionParams {
+  phoneNumber?: string;
+  callerId?: string;
+  callerIdName?: string;
+  callFromQueueId?: string;
+  callQueueId?: string;
+  callUserId?: string;
+  priority?: number;
+  languageId?: string;
+  routingSkillsIds?: string[];
+  conversationIds?: string[];
+  participants?: ISdkSoftphoneDestination[];
+  uuiData?: string;
+}
+```
+
+* phoneNumber?: string - Optional: The phone number to dial.
+* callerId?: string - Optional: The caller id phone number for outbound call.
+* callerIdName?: string - Optional: The caller id name for outbound call.
+* callFromQueueId?: string - Optional: The queue id to place the call on behalf of.
+* callQueueId?: string - Optional: The queue id to call.
+* callUserId?: string - Optional: The user id to call.
+* priority?: number - Optional: The priority of the call.
+* languageId?: string - Optional: The language skill id to use for routing call if calling a queue.
+* routingSkillsIds?: string[] - Optional: The routing skills ids to use for routing call if calling a queue.
+* conversationIds?: string[] - Optional: List of existing conversations to merge into new ad-hoc conference.
+* participants?: ISdkSoftphoneDestination[] - Optional: List of participants to add to the call if starting a conference call.
+* uuiData?: string - Optional: User to user information managed by SIP session app.
```

Comment:
NIT: maybe wrap `uuiData?: string` in backticks so it get the monospaced formatting.